uniswap-v2-core/contracts/UniswapV2Pair.sol

```diff
@@ -35,6 +47,27 @@ contract UniswapV2Pair is IUniswapV2Pair, UniswapV2ERC20 {
         unlocked = 1;
     }
 
+    modifier onlyFactory() {
+        require(msg.sender == factory, 'Vexchange: FORBIDDEN');
+        _;
+    }
+
+    function setRecoverer(address _recoverer) external onlyFactory {
+        recoverer = _recoverer;
+    }
+
+    function setPlatformFee(uint _platformFee) external onlyFactory {
```

Comment:
Should these be `_variable` given external method? Maybe drop the `_`

Reply:
I think this is consistent with the rest of uniswap (apply _ only to provide uniqueness when required) - so I'd suggest we leave as-is for the moment.